### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mericstam**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 - Class Date: April 13, 2021
 - Facilitator: @gusshawstewart
 
-## Talk to Us!!
+## Talk to Us!!!
```
Comment (mericstam) [inline README.md:6]: not sure it is good. what do you think?

Comment (gusshawstewart) [inline README.md:6]: This is a good change!

File: README.md [hunk 2]
```diff
@@ -3,7 +3,7 @@
 - Class Date: April 13, 2021
```
Comment (gusshawstewart) [inline README.md]: have you thought about doing it like this:
```suggestion
- Class Date: April 13, 2025
```

**Review by gusshawstewart — APPROVED** [review]
This good to me! 😃

**Pull request merged**